src/simplewallet/simplewallet.cpp

```diff
@@ -6498,6 +6498,7 @@ void simple_wallet::check_for_inactivity_lock(bool user)
 #endif
     tools::clear_screen();
     m_in_command = true;
+    int loop_counter = 0;
```

Comment:
A small nitpick, but unlike in C, where you're forced to declare the variables at the beginning of the scope, in higher level languages it's possible and expected to declare variables as close as their usage as possible (if it doesn't lead to multiple excess computations that is).

So:

```C++
int loop_counter = 0;
while (/*...*/
```